clients/fides-js/src/lib/initialize.ts

```diff
@@ -458,30 +457,25 @@ export const initialize = async ({
          * set any applicable notices to "opt-out" unless the user has previously
          * saved consent, etc.
          *
-         * NOTE: Do *not* await the results of this function, even though it's
-         * async and returns a Promise! Instead, let the GPC update run
-         * asynchronously but continue our initialization. If GPC applies, this
-         * will kick off an update to the user's consent preferences which will
-         * also call the Fides API, but we want to finish initialization
-         * immediately while those API updates happen in parallel.
+         * NOTE: We want to finish initialization immediately while GPC updates
+         * run in parallel. To ensure that any GPC API calls don't block the rest
+         * of the code from executing, we use setTimeout with no delay which
+         * simply moves it to the end of the JavaScript event queue.
```

Comment:
ah interesting hack, I didn't know about this. So this means the JS runtime will wait until the call stack is empty (meaning it has completed running the other init logic in this file), then run `automaticallyApplyGPCPreferences` at the very end? What exactly do you mean by `while GPC updates run in parallel.`?

Reply:
Yes, that's an accurate description.

"...in parallel" is a phrase I borrowed from Neville's prior notes that I removed here. I'll update to say "...in the background" which I think is a little more clear, don't you?